packages/api/lib/writeRecords/write-granules.js

```diff
@@ -596,7 +619,7 @@ const _writeGranule = async ({
   writeConstraints = true,
 }) => {
   const { status } = apiGranuleRecord;
-  const pgGranule = await _writeGranuleRecords({
+  const pgGranuleResult = await _writeGranuleRecords({
     apiGranuleRecord,
```

Comment:
If this has been removed from the parameters object we need to remove it here too.

Reply:
I'm not sure I understand what you're saying we need to remove, are you referring to `apiGranuleRecord`? If so, that was not removed in this method.